tests/unit/test_events.py

```diff
@@ -84,6 +86,18 @@ def github_installation(self, request):
         with open(DATA_DIR / "webhooks" / "github" / file) as outfile:
             return json.load(outfile)
 
+    @pytest.fixture()
+    def merge_request(self):
+        with open(DATA_DIR / "webhooks" / "gitlab" / "mr_event.json", "r") as outfile:
+            return json.load(outfile)
+
+    @pytest.fixture()
+    def merge_request_update(self):
+        with open(
+            DATA_DIR / "webhooks" / "gitlab" / "mr_update_event.json", "r"
+        ) as outfile:
+            return json.load(outfile)
+
```

Comment:
Regarding the mock config:

Instead of using this:

```python
    @pytest.fixture()
    def mock_config(self):
        service_config = ServiceConfig()
        service_config.github_app_id = 123123
        service_config.github_app_cert_path = None
        service_config.github_token = "token"
        service_config.dry_run = False
        service_config.github_requests_log_path = "/path"
        ServiceConfig.service_config = service_config
```

You can use this:

```python
    @pytest.fixture()
    def mock_config(self):
        service_config = ServiceConfig()
        service_config.services = {
            GithubService(token="token"),
            GitLabService(token="token"),
        }
        service_config.dry_run = False
        service_config.github_requests_log_path = "/path"
        ServiceConfig.service_config = service_config
```